src/client/orderbook/OrderbookFullClient.ts

```diff
@@ -201,6 +210,31 @@ export class OrderbookFullClient extends OrderbookReadClient {
     };
   }
 
+  public async listMultipleNftsWithCommission(p: {
+    beneficiary: SuiAddress;
+    commissions: number[];
```

Comment:
Might be good to add a check that commissions, prices and nfts are all equal length

Reply:
Added a runtime check, lmk if you are about interface.